filterdata/filter.C

```diff
@@ -0,0 +1,46 @@
+#include "FilterTree4ML.cc"
```

Comment:
The `printTree.C` macro is not committed, so this macro would not work. I would suggest you either to commit also the `printTree.C` macro or instead to remove this macro too from the PR if you don't want to commit it.